cmd/notary/main.go

```diff
@@ -1,96 +1,71 @@
 package main
 
 import (
-	"crypto/x509"
 	"fmt"
 	"os"
 	"os/user"
-	"path"
-	"time"
+	"path/filepath"
 
 	"github.com/Sirupsen/logrus"
 	"github.com/spf13/cobra"
 	"github.com/spf13/viper"
 
-	"github.com/docker/notary/trustmanager"
+	"github.com/docker/notary/pkg/passphrase"
 )
 
 const configFileName string = "config"
-const configPath string = ".docker/trust/"
-const trustDir string = "trusted_certificates/"
-const privDir string = "private/"
-const rootKeysDir string = "root_keys/"
+const defaultTrustDir string = ".notary/"
```

Comment:
Any proposals on what to do?

People could add a configuration file to config.json under ~./notary to always point it to ~/.docker/trust